### PR TITLE
File added : Assignment 5 & 6 Added

### PR DESCRIPTION
**Pull request by Bhavika222**
ok

### CODE REVIEW

File: Bhavika/html/Assignment5/5ASS.html [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+        }
+        ul{
+            list-style-type: square;
+        }
+        a:visited {
+             color: green
+        }
+    </style>
+</head>
+    <body>
```
Comment (Pranjali1999) [inline Bhavika/html/Assignment5/5ASS.html]: body is exact below the closing head tag.

File: Bhavika/html/Assignment5/5ASS.html [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+                 My Hobbies are Listening to music, cooking, dancing,and reading books.<br>
+                 That's all about me,Thankyou again  for this opportiunity.
+                </p>
+    <h2>More About Me</h2>
+                <ul class="para">
+                  <li><a href="a" target=" _blank">My Resume</a></li>
+                  <li><a href="b" target="_blank">My Hobbies</a></li>
+                  <li><a href="c" target="_blank">Photo Gallery</a></li>
+                </ul>
+</head>
```
Comment (Pranjali1999) [inline Bhavika/html/Assignment5/5ASS.html]: Head is already closed at line number 40, why did you close again?
& give proper title name.

File: Bhavika/html/Assignment5/5ASS.html [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+    <body>
+      <h1> Bhavika Madrap</h1>
+      <div class="bgimage"></div>
+    <h2>About Me</h2>
+         <p class="para"> My name is Bhavika Bhupal Madrap.I am From Amravati . but currently Staying in Pune.
+             I did My graduation in BCA from SGBAU University.
+             Currently I am pursing MCA in Savitribai Phule Pune University.<br>
+             I belong to a nuclear family. My strength are I am self Motivated,hardworking and a disciplined person my short term goal
+             is to get a job in a reputed company and my long term goal is to achive a good position Where I can build my career and help organization too.<br>
+                 As a fresher I am interested in learning new things.But I am waiting fo an opportunity to implement them in a pratical way.
```
Comment (Pranjali1999) [inline Bhavika/html/Assignment5/5ASS.html]: All line of paragraph should be starting from same point.

File: Bhavika/html/Assignment6/Ass6.html [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+        <label for="lname">Last name:</label>
+        <input type="text" id="lname" name="lname" /><br />
+        <br />
+        <label for="Gender">Gender:</label>
+        <input type="radio" id="male" name="Gender" value="male" />
+        <label for="male">Male</label>
+        <input type="radio" id="female" name="gender" value="female" />
+        <label for="female">Female</label>
+        <input type="radio" id="other" name="gender" value="other" />
+        <label for="other">other</label><br /><br />
```
Comment (Pranjali1999) [inline Bhavika/html/Assignment6/Ass6.html:47]: Why did you use the br tag twice?

**Review by Pranjali1999 — CHANGES_REQUESTED** [review]

File: Bhavika/html/Assignment5/Assig5.html [hunk 5]
```diff
@@ -0,0 +1,63 @@
...
+        }
+        ul{
+            list-style-type: square;
+        }
+        a:visited {
+             color: green
+        }
+    </style>
+</head>
+    <body>
```
Comment (Priyankashelke2411) [inline Bhavika/html/Assignment5/Assig5.html]: give proper indentations

**Review by Priyankashelke2411 — CHANGES_REQUESTED** [review]

File: Bhavika/html/Assignment5/Assig5.html (hunk 5 above)
Comment (Bhavika222) [reply Bhavika/html/Assignment5/Assig5.html]: Mam Plz check my new uploaded file.
 https://github.com/Pawan1005/MI-11-MEAN/pull/272/commits/6d81e6bd5a6e8b2b0b6254960918d3c6839075aa

**Review by Bhavika222 — COMMENTED** [review]
Mam Plz check my updated file.
https://github.com/Pawan1005/MI-11-MEAN/pull/272/commits/6d81e6bd5a6e8b2b0b6254960918d3c6839075aa

**Review by Priyankashelke2411 — APPROVED** [review]

**Review by Priyankashelke2411 — APPROVED** [review]

**Review by Akash-ing — APPROVED** [review]

File: Bhavika/html/Assignment6/Ass6.html [hunk 6]
```diff
@@ -0,0 +1,66 @@
...
+        <style>
+            body {
+                font-family: Arial, Helvetica, sans-serif;
+                background-color: #e2dfdc;
+            }
+            h1 {
+                border: 2px solid #91b1c5;
+                background-color: #91b1c5;
+            }
+            h1 {
```
Comment (KomalShinde813) [inline Bhavika/html/Assignment6/Ass6.html]: combine this in above h1 tag style

File: Bhavika/html/Assignment6/Ass6.html [hunk 7]
```diff
@@ -0,0 +1,66 @@
...
+                text-align: right;
+                padding: 10px;
+            }
+            h2 {
+                background-color: #91b1c5;
+            }
+            h2 {
```
Comment (KomalShinde813) [inline Bhavika/html/Assignment6/Ass6.html]: combine this in above h2 tag style

File: Bhavika/html/Assignment6/Ass6.html [hunk 8]
```diff
@@ -0,0 +1,66 @@
...
+        <h1>MI-9 Intern HTML Assesment</h1>
+        <h2><i>Personal Details</i></h2>
+        <label for="fname">First name:</label>
+        <input type="text" id="fname" name="fname" /><br />
+        <br />
+        <label for="lname">Last name:</label>
+        <input type="text" id="lname" name="lname" /><br />
+        <br />
+        <label for="Gender">Gender:</label>
+        <input type="radio" id="male" name="Gender" value="male" />
```
Comment (KomalShinde813) [inline Bhavika/html/Assignment6/Ass6.html]: check this really working or not may be its selected male as well as other options too

File: Bhavika/html/Assignment6/Ass6.html [hunk 9]
```diff
@@ -0,0 +1,66 @@
...
+        <label for="male">Male</label>
+        <input type="radio" id="female" name="gender" value="female" />
+        <label for="female">Female</label>
+        <input type="radio" id="other" name="gender" value="other" />
+        <label for="other">other</label><br /><br />
+        <label for="Age">Age:</label>
+        <input type="radio" id="less than 18" name="less than 18" value="less than 18" />
```
Comment (KomalShinde813) [inline Bhavika/html/Assignment6/Ass6.html]: name should be same in radio buttons otherwise we can choose multiple options

**Review by KomalShinde813 — CHANGES_REQUESTED** [review]

**Review by vishnukopnar — APPROVED** [review]

**Review by Pranjali1999 — COMMENTED** [review]
Give proper title name

**Review by KomalShinde813 — APPROVED** [review]

**Review by Pranjali1999 — APPROVED** [review]

**Pull request merged**